models/sessions/scratch/bigquery/snowplow_web_sessions_this_run.sql

```diff
@@ -302,7 +359,13 @@ select
         {%- for conv_def in var('snowplow__conversion_events') %}
     {{ snowplow_web.get_conversion_columns(conv_def, names_only = true)}}
         {%- endfor %}
+    {% if var('snowplow__total_all_conversions', false) %}
```

Comment:
Should we add `var('snowplow__conversion_events')` to the `if` statement?

Reply:
It's already nested within that if block